cppForSwig/Makefile

```diff
@@ -17,7 +17,14 @@ endif
 platform=$(shell uname)
 
 ifeq ($(shell uname), Darwin)
+# XCode 8 breaks when compiling for 10.7, XCode 7 and below does not
+XCODE_VERSION:=$(shell xcodebuild -version | awk '{ print $$2 }' | head -n1)
+ifeq ($(shell echo "${XCODE_VERSION} < 8" | bc), 0)
```

Comment:
Actually, it's fine as written - notice it's comparing to `0`. That's obviously a bit confusing though so I've switched it.

As for the parse error, I couldn't believe I would have shipped this broken then realized the latest XCode version is `8.2.1` (at the time it was `8.1`) - `8.2.1` is not a valid float and thus `bc` calls it an error.